### DISCUSSION TITLE
DB Engine Error: issue 1011- superset encountered an unexpected error. Invalid decryption key.

### DISCUSSION THREAD

**Discussion by waleedbamber** [apache/superset#27247] [Q&A / Help]
I created some dashboards using datasets from PostgreSQL. After the dashboards were finalized and published, i stopped superset and turned off my laptop. when ran superset the next day, the graphs in my dashboard do not open and it says db engine error. Usually, when I run apache superset, I use these commands (The OS I use is Linux):  
 •	Create a virtual environment by running: “python3 –m venv supersetvenv”.

•	Activate the virtual environment with the command: “source supersetvenv/bin/activate”.

•	Set the flask app by executing: “export FLASK_APP=super

•	Generate a new security key using: “openssl rand -base64 42”

•	Export the generated security key:  “export SUPERSET_SECRET_KEY=<key>”. (Replace <key> with the generated key)

•	Launch Apache-Superset by running: “superset run -p 8099 --with-threads --reload --debugger”.

![pic1](https://github.com/apache/superset/assets/86602070/6046de93-6c52-4aad-9bbf-56b0c0c8aa76)
![pic2](https://github.com/apache/superset/assets/86602070/ad4e0fe0-4464-4a19-a11d-2f2779202d49)
![pic3](https://github.com/apache/superset/assets/86602070/4187ee2d-c6a9-4744-bc44-40a56c835091)

**Comment by a-erkinov** [comment]
I'm also having a similar issue.  In my case I changed my dataset from oracle to clickhouse. 
![image](https://github.com/user-attachments/assets/68cdc04c-dd1d-428a-8ce1-a0b044327d07)
